pages/result.tsx

```diff
@@ -1,12 +1,48 @@
-export const Result = () => {
+import Image from "next/image";
+import styled from "styled-components";
+import { LinkedButton } from "./components/LinkedButton";
+
+type ResultTypes = {
+  calculatedTime: number;
+  minutes: number;
+  seconds: number;
+};
+
+const Result = ({ minutes, seconds }: ResultTypes) => {
   return (
     <>
       <h1>Glückwunsch, du bist ein(e) Schnellleser(in)!</h1>
```

Comment:
How many l's do you want in "Schnellleser(in)"?

Yes.

Reply:
I'd love to add about a dozen, since "l" is a very pretty letter. Unfortunately, the number of "l"'s is restricted by the German [Duden](https://www.duden.de/rechtschreibung/Schnellleser).
What a pity!